### ISSUE TITLE
running bin\plugin list on a newly unpacked release on windows fails

### ISSUE THREAD

**Issue by jsvd** [elasticsearch/logstash#2641]
I generated a new release of logstash with rake artifact:tar and moved it to windows, unpacked and ran:

```
C:\test\logstash-2.0.0.dev>bin\plugin list
Bundler::GemNotFound: Could not find rake-10.4.2 in any of the sources
      materialize at C:/test/logstash-2.0.0.dev/vendor/bundle/jruby/1.9/gems/bun
dler-1.8.2/lib/bundler/spec_set.rb:92
             map! at org/jruby/RubyArray.java:2444
      materialize at C:/test/logstash-2.0.0.dev/vendor/bundle/jruby/1.9/gems/bun
dler-1.8.2/lib/bundler/spec_set.rb:85
            specs at C:/test/logstash-2.0.0.dev/vendor/bundle/jruby/1.9/gems/bun
dler-1.8.2/lib/bundler/definition.rb:132
        specs_for at C:/test/logstash-2.0.0.dev/vendor/bundle/jruby/1.9/gems/bun
dler-1.8.2/lib/bundler/definition.rb:177
  requested_specs at C:/test/logstash-2.0.0.dev/vendor/bundle/jruby/1.9/gems/bun
dler-1.8.2/lib/bundler/definition.rb:166
  requested_specs at C:/test/logstash-2.0.0.dev/vendor/bundle/jruby/1.9/gems/bun
dler-1.8.2/lib/bundler/environment.rb:18
            setup at C:/test/logstash-2.0.0.dev/vendor/bundle/jruby/1.9/gems/bun
dler-1.8.2/lib/bundler/runtime.rb:13
            setup at C:/test/logstash-2.0.0.dev/vendor/bundle/jruby/1.9/gems/bun
dler-1.8.2/lib/bundler.rb:122
   bundler_setup! at C:/test/logstash-2.0.0.dev/lib/logstash/environment.rb:66
           (root) at C:\test\logstash-2.0.0.dev\lib\logstash\runner.rb:8
```

This is because ".bundle/config":
```
---
BUNDLE_PATH: /Users/joaoduarte/projects/logstash/vendor/bundle
```
contains the bundle path used during the generation of the release.

Fix proposal:

Before generating the artifact, modify ".bundle/config" to have a relative path for BUNDLE_PATH:

```
---
BUNDLE_PATH: vendor/bundle
```

[EDIT] Alternative fix is to NOT include the ".bundle/config" file in the release

**Comment by jsvd** [comment]
after fixing this, the following error occurs:

```
.....
logstash-patterns-core (0.1.5)
TypeError: cannot convert instance of class org.jruby.RubyArray to int
    exit at C:/test/logstash-2.0.0.dev/lib/logstash/program.rb:10
    main at C:\test\logstash-2.0.0.dev\lib\logstash\runner.rb:86
  (root) at C:\test\logstash-2.0.0.dev\lib\logstash\runner.rb:210
```

**Comment by jsvd** [comment]
fixed in cc6f8473ecb7097e772feda1948526bd0f90404a
